### DISCUSSION TITLE
How to trigger CSS animations

### DISCUSSION THREAD

**Discussion by tgd-git** [cables-gl/cables_docs#265] [Answers & Questions] [answered]
hey i have a DivElement op and a CSS op. Can i somhow trigger the animation externally to play it forward and backward?  
 
`my CSS contents with simple animation example:`
```
div.AnimatedDiv{
  animation-name: example;
  animation-duration: 3s;  
  animation-fill-mode: both;   
}
@keyframes example {
  from {top: 0px;}
  to {top: 200px;}
}
```

**Comment by steam0r** [comment, marked as answer]
you can use https://cables.gl/op/Ops.Html.ToggleClass to add and remove classes from DivElement.

you can also use https://cables.gl/op/Ops.Html.CSSProperty_v2 or https://cables.gl/op/Ops.Html.CSSPropertyString to control the animation part from cables timers and triggers.

hope that helps!

**Reply by tgd-git** [reply]
yes i was fiddeling with these later and had a way kind of working, it felt not right to me though. i thought there might be something more specific to the task i was not finding. also i am not super into html and css so i might just misunderstand something. Thanks anyways!
